src/sass/app.scss

```diff
@@ -1,3 +1,5 @@
+@import './variables.scss';
```

Comment:
Good start to making our css better!!!

Reply:
Something to consider (not necessary for this PR but I think would be nice for at least @floraguolr to continue on with): Let's use scss modules so that we can more explicitly trace what files css classes come from. See https://blog.bitsrc.io/how-to-use-sass-and-css-modules-with-create-react-app-83fa8b805e5e